### DISCUSSION TITLE
Bravo ! very good work. I just want to know where shoul i put the motions models and Motion lora ( from animate diff) in wich folder?

### DISCUSSION THREAD

**Discussion by Sinclou** [painebenjamin/app.enfugue.ai#111] [Q&A]
Animate diff ckpt folder

**Comment by painebenjamin** [comment]
Hello!

Thank you for using Enfugue and the kind words! 

The default directory for LoRA of all kinda (including motion) is at `~/.cache/enfugue/lora`, where `~` is your home directory (`/home/<username>` on Linux/MacOS or `C:\Users\<username>` on Windows.) The default directory for motion modules is similar, at `~/.cache/enfugue/motion`.

Thank you again!

**Comment by Sinclou** [comment]
Thank you very much!!Mathieu BFClapmusicLe 19 nov. 2023 à 20:15, Benjamin Paine ***@***.***> a écrit :﻿
Hello!
Thank you for using Enfugue and the kind words!
The default directory for LoRA of all kinda (including motion) is at ~/.cache/enfugue/lora, where ~ is your home directory (/home/<username> on Linux/MacOS or C:\Users\<username> on Windows.) The default directory for motion modules is similar, at ~/.cache/enfugue/motion.
Thank you again!
